README.md

```diff
@@ -594,6 +594,20 @@ second entry, the error structure should look like this:
 }
 ```
 
+Additionally, you can assign errors to a field using your own functions.
+
+```js
+this.formState = form.state(o, {
```

Comment:
Hm, this is a bit misleading, as it makes it seem like the getError hook is primarily about validation, but it shouldn't really be used for validation (we already have validation for that). I'd rather have an example that uses .path.

Reply:
Used .path in the example